apis/core/lib/domain/column-mapping/update.ts

```diff
@@ -50,10 +50,12 @@ export async function updateColumnMapping({
   }
 
   if (targetProperty.value !== columnMapping.targetProperty.value) {
-    const mappingExists = await findMapping(table, targetProperty, store)
-    if (mappingExists) {
-      const column = await store.getResource<CsvColumn>(mappingExists.sourceColumn?.id)
-      throw new DomainError(`Target property already mapped from column ${column?.name}`)
+    if (table.isObservationTable) {
+      const mappingExists = await findMapping(table, targetProperty, store)
+      if (mappingExists) {
+        const column = await store.getResource<CsvColumn>(mappingExists.sourceColumn?.id)
+        throw new DomainError(`Target property already mapped from column ${column?.name}`)
+      }
     }
 
     if (table.types.has(cc.ObservationTable)) {
```

Comment:
If we check for the observation table, I would merge both checks for the observation tables , no?
```suggestion
    if (table.isObservationTable) {
      const mappingExists = await findMapping(table, targetProperty, store)
      if (mappingExists) {
        const column = await store.getResource<CsvColumn>(mappingExists.sourceColumn?.id)
        throw new DomainError(`Target property already mapped from column ${column?.name}`)
      }
```

Reply:
Be my guest ;) I don't have much time to work on this. I just noticed that the behavior was not consistent between "create" and "update" and quickly wanted to fix it

Reply:
Here you go :wink: